Probesys/bileto
# tec: Upgrade dependencies

Changes proposed in this pull request:

- (misc) Change the profile icon
- Add documentation about the dependencies
- Upgrade frontend and backend dependencies

How to test the feature manually:

1. Use the app, check everything works as before
2. Check the profile icon has changed

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
